From 98f1556d8a134f54d62ebdac27e9d16aa7884983 Mon Sep 17 00:00:00 2001
From: "Brian C. Lane" <bcl@redhat.com>
Date: Tue, 20 Mar 2012 17:17:10 -0700
Subject: [PATCH 2/2] tests: add t0301-overwrite-gpt-pmbr.sh

Make sure parted checks the PMBR before the GPT partition table.

* NEWS: Update with new GPT behavior
* tests/Makefile.am: Add new test
* tests/overwrite-gpt-pmbr.sh: new test
---
 NEWS |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/NEWS b/NEWS
index fe0fcdd..b0a0657 100644
--- a/NEWS
+++ b/NEWS
@@ -1,5 +1,12 @@
 GNU parted NEWS                                    -*- outline -*-
 
+* Noteworthy changes in release 3.1-2 (2012-03-21) [Fedora]
+
+** Bug Fixes
+
+  libparted: Treat disks without a PMBR as msdos labeled disks
+  even if they have GPT partition tables.
+
 * Noteworthy changes in release 3.1 (2012-03-02) [stable]
 
 ** New features
-- 
1.7.7.6

